pyega3/pyega3.py

```diff
@@ -8,11 +8,13 @@
 import logging
 import math
 import os
+import platform
 import random
 import sys
 import time
 
 import htsget
+import psutil
```

Comment:
When I tested running pyega3 from this branch, I had to manually install `psutil` even after executing `osx_dependency_install.sh` to install dependencies. I checked and I don't see a command to install `psutil` in this script. In order to support usage of pyega3 if someone installs from GH (rather than using `pip3` or `conda`), can you please be sure that any new dependencies are added to the `*dependency_install.sh` scripts?

Reply:
Oh, I missed adding the dependency in other files. Good to catch it. I added it now. Hopefully should work.

Reply:
Dependency install work for psutil using the `osx_dependency_install.sh` script - thanks!